src/EcsRx/Collections/EntityCollectionManager.cs

```diff
@@ -119,7 +119,7 @@ public IEnumerable<IEntity> GetEntitiesFor(IGroup group, int collectionId = Pool
             if(group is EmptyGroup)
             { return new IEntity[0]; }
 
-            if (collectionId == PoolLookups.NoPoolDefined)
```

Comment:
Hey, you are right, there is an issue here but I think the fix is to use the if on the lower bit of logic. As this should basically check if you have a collectionId, if so use that collectionId, if not then it should just scan everything.

In the proposed change it would scan everything unless the collectionId matched the default one (which I think is 1), so that would never scan the collection you want. I will push up a change shortly, also the `PoolLookups` should probably be changed to `EntityLookups.DefaultCollectionId` etc, as historically the `EntityCollection` objects used to be `Pool` objects.